Formula/mrboom.rb

```diff
@@ -24,21 +24,8 @@ def install
   end
 
   test do
-    require "pty"
-    require "expect"
-    require "timeout"
-    PTY.spawn(bin/"mrboom", "-m", "-f 0", "-z") do |r, _w, pid|
-      sleep 1
-      Process.kill "SIGINT", pid
-      assert_match "monster", r.expect(/monster/, 10)[0]
-    ensure
-      begin
-        Timeout.timeout(10) do
-          Process.wait pid
-        end
-      rescue Timeout::Error
-        Process.kill "KILL", pid
-      end
-    end
+    # gui app
+    assert_match "Set joysticks dead zone, default is 8000",
+      shell_output("#{bin}/mrboom --help")
```

Comment:
Why are we removing a test in favor of a help text?

Reply:
I think that is a GUI app, that is why I updated the test back to a simpler version.

Reply:
Did the old test not test the app? Because otherwise I think that one should be fixed instead.

Reply:
with mrboom 5.2, let me see if the old test works fine or not.